Additional structural edits for vaccines manuscript (#1073)
greenelab/covid19-review: content/22.vaccines.md

### Abstract

Vaccines have revolutionized the relationship between people and disease.
In the 21^st^ century, several emergent viruses have emphasized the importance of rapid and scalable vaccine development programs.
During the current pandemic caused by _Severe acute respiratory syndrome coronavirus 2_ (SARS-CoV-2), recent biotechnological advances in vaccine design provided the circumstances for the development and deployment of vaccines at an unprecedented pace.
The genome sequence of SARS-CoV-2 was released on January 10^th^, 2020, allowing for global efforts in vaccine development to begin within two weeks of the international community becoming aware of the new viral threat.

> Both established vaccine platforms and newer technologies have been explored against SARS-CoV-2.

Author: ```suggestion
Both established vaccine platforms and more recent technologies have been explored against SARS-CoV-2.
```

The development of vaccines is widely considered one of the most important medical advances in human history.
The past 150 years have seen a rapid diversification in the approaches of vaccine development available [@doi:10.1073/pnas.1400472111].
Since the turn of the millennium, particular interest has emerged in the potential to develop vaccines as a rapid response to emerging viral threats.
Such threats have included severe acute respiratory syndrome (SARS), "swine flu" (H1N1), Middle East respiratory syndrome (MERS), and Ebola virus disease (EVD), all of which have underscored the importance of a rapid global response to a new infectious virus.

> However, vaccine development has historically been slow, and because vaccines fail to provide immediate prophylactic protection or to treat ongoing infections, their application to most of these epidemics has been limited [@doi:10.3390/v10120680].

Author: ```suggestion
However, vaccine development has historically been slow, and because vaccines fail to provide immediate prophylactic protection or treatment of ongoing infections, their application to most of these epidemics has been limited [@doi:10.3390/v10120680].
```

On the other hand, the pandemic caused by _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) has highlighted a confluence of circumstances that positioned vaccine development as a key player in efforts to control the virus and mitigate its damage.
Examining the vaccine development programs tackling the COVID-19 pandemic alongside other 21^st^ century efforts to control emerging viral threats demonstrates the significant biotechnological advances in this field.

> In most recent epidemics, vaccine development has not occurred rapidly enough to change the course of the disease's spread.

Author: ```suggestion
In recent epidemics, vaccine development has not occurred rapidly enough to change the course of the disease's spread.
```

> One of the more successful programs recently was the development of a vaccine against H1N1 influenza, which benefited from the strong existing infrastructure for influenza vaccines, along with the fact that regulatory agencies had already determined that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change [@doi:10.1056/NEJMp2005630].

Author: ```suggestion
Indeed, one of the more successful recent programs was the development of a vaccine against H1N1 influenza, which benefited from the strong existing infrastructure for influenza vaccines, along with the fact that regulatory agencies had already determined that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change [@doi:10.1056/NEJMp2005630].
```

In most recent cases, vaccine development was still in progress when overall control of the virus was achieved.
Technologies such as inactivated viral vaccines, live attenuated viral vaccines, protein subunit vaccines, and recombinant vector-based vaccines were explored for SARS [@doi:10.3201/eid1107.050219; @doi:10.1038/nrmicro3143], although the epidemic was controlled before a vaccine became available [@doi:10.1056/NEJMp2005630].
Additionally, concerns arose about whole-virus and some protein subunit vaccines against SARS-CoV-1 potentially inducing a pathological immune response [@doi:10.1016/j.micinf.2014.05.002; @doi:10.1371/journal.pone.0035421].
DNA vaccine development efforts also began, but did not proceed past animal testing [@doi:10.1038/nrmicro3143].
Similarly, viral vector, protein subunit, and DNA vaccines were explored for MERS-CoV, but outbreaks are sporadic and difficult to predict, making vaccine testing and the development of a vaccination strategy difficult [@doi:10.1080/21645515.2017.1358325].

> Likewise, the development of viral-vectored Ebola virus vaccines was undertaken, but the pace of vaccine development was behind the spread of the virus from early on [@doi:10.1126/science.345.6203.1441], and although the candidate Ebola vaccine V920 showed promise in preclinical and clinical testing, it did not receive breakthrough therapy designation until summer 2016, when the Ebola outbreak was winding down [@doi:10.1016/j.vaccine.2017.05.097].

Author: ```suggestion
Likewise, the development of viral-vectored Ebola virus vaccines was undertaken, but the pace of vaccine development was behind the spread of the virus from early on [@doi:10.1126/science.345.6203.1441].
Although the candidate Ebola vaccine V920 showed promise in preclinical and clinical testing, it did not receive breakthrough therapy designation until the summer of 2016, by which point the Ebola outbreak was winding down [@doi:10.1016/j.vaccine.2017.05.097].
```

The approach was (debatably) the first example of a live-attenuated virus being used to induce immunity [@doi:10.1016/j.virol.2015.03.032; 10.1073/pnas.1400472111].
Many subsequent efforts to develop live-attenuated viral vaccines relied on either the identification of related zoonotic viruses that are less virulent in humans (e.g., cowpox/horsepox or rotavirus vaccines) or efforts to attenuate the virus through culturing it _in vitro_ [@doi:10.1073/pnas.1400472111; @doi:10.1377/hlthaff.24.3.611].
Such approaches still carry significant risks, however [@doi:10.1073/pnas.1400472111].

Efforts to overcome the limitations of live-virus vaccines led to the development of approaches to inactivate viruses (circa 1900) and to purify proteins from viruses cultured in eggs (circa 1920) [@doi:10.1073/pnas.1400472111; @doi:10.1073/pnas.1402981111]. <!--To do: remove proteins from this sentence-->
Inactivated viral vaccines still raised some concerns about safety, including that back-mutations could potentially lead the inactivated vaccines to become virulent or that recombination could occur between the inactivated virus and other viruses in circulation [@doi:10.3389/fimmu.2019.00594].

> For example, in the famous 1955 Cutter Incident, errors in the manufacturing process produced polio vaccines containing live polio virus, leading to an outbreak in the United States [@doi:10.1056/NEJMp048180].

Reviewer: ```suggestion
For example, in the famous 1955 Cutter incident, errors in the manufacturing process produced polio vaccines containing live polio virus, leading to an outbreak in the United States [@doi:10.1056/NEJMp048180].
```

> For example, in the famous 1955 Cutter Incident, errors in the manufacturing process produced polio vaccines containing live polio virus, leading to an outbreak in the United States [@doi:10.1056/NEJMp048180].

Author: ```suggestion
For example, in the famous 1955 Cutter incident, errors in the manufacturing process produced polio vaccines containing live polio virus, leading to an outbreak in the United States [@doi:10.1056/NEJMp048180].
```

Vaccine efficacy against cases caused by VOC, among which the Alpha variant was predominant (88.6%), was reported to be 92.6% [@doi:10.1101/2021.10.05.21264567]. 

##### Virus-Like Particle

<!--To Do: add info about VLP and phase 1/2 trials:
https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker
https://pubmed.ncbi.nlm.nih.gov/20923267/
https://jnanobiotechnology.biomedcentral.com/articles/10.1186/s12951-021-00806-7
-->

##### Recombinant Protein Vaccine

<!--To Do: add, see https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker-->

#### Nucleic Acid Vaccines

While traditional methods of vaccine development such as inactivated whole viruses are still used today (Figure @fig:vaccines), biomedical research in the 21^st^ century has been significantly influenced by the genomic revolution, and vaccine development is no exception.
The shift towards omics-based approaches to vaccine development began to take hold with the development of the meningococcal type B vaccine using reverse vaccinology in the early 2010s [@doi:10.7774/cevr.2014.3.1.1; @doi:10.1111/j.1469-0691.2012.03939.x].

> In this way, the Genomic Revolution catalyzed a fundamental shift in the development of vaccines.

Reviewer: ```suggestion
In this way, the genomic revolution catalyzed a fundamental shift in the development of vaccines.
```

> In this way, the Genomic Revolution catalyzed a fundamental shift in the development of vaccines.

Author: ```suggestion
In this way, the genomic revolution catalyzed a fundamental shift in the development of vaccines.
```
